src/view/components/common/toolbar-notifications/toolbar-notifications.ts

```diff
@@ -0,0 +1,50 @@
+import {Component, ViewEncapsulation} from '@angular/core';
+import {MdIcon} from '@angular2-material/icon/icon';
+import {MdButton} from '@angular2-material/button/button';
+import {MD_CARD_DIRECTIVES} from '@angular2-material/card/card';
+import {INotification, NotificationsService} from '../../../../api/services/notification-service';
+import {NotificationsList} from '../notifications/notifications';
+
+@Component({
+    moduleId: __moduleName, // REQUIRED to use relative path in styleUrls
+    selector: 'dot-toolbar-notifications',
+    styleUrls: ['toolbar-notifications.css'],
+    template: `
+    <div class="dot-notifications">
+        <button md-icon-button (click)="toggleNotifications()">
+            <md-icon class="md-24">notifications</md-icon>
+        </button>
+        <md-card *ngIf="showNotifications" class="notifications">
+            <md-card-title>Notifications</md-card-title>
+            <md-card-content>
+                <dot-notifications-list [notifications]="notifications"></dot-notifications-list>
+            </md-card-content>
+        </md-card>
+    </div>
+    `,
+    providers: [NotificationsService],
+    directives: [MdIcon, MdButton, NotificationsList, MD_CARD_DIRECTIVES],
+    encapsulation: ViewEncapsulation.Emulated
+})
+export class ToolbarNotifications {
```

Comment:
This is acting as a "container components" as you can see, it will fetch the data and pass it to the "dumb" notification component.